docs/DeveloperGuide.md

```diff
@@ -240,18 +240,18 @@ How the `Logic` component works:
 
 Figure 7 below illustrates the interactions within the `Logic` component for the `execute("delete-index 1")` API call.
```

Comment:
The description is for `delete-index 1` but your sequence diagram is for `delete 1-2`. Shall we fix it to be the same?

Reply:
The sequence diagram shows `delete-index 1 2`